src/Stratis.SmartContracts.IntegrationTests/SmartContractMemoryPoolTests.cs

```diff
@@ -99,6 +99,13 @@ public void SmartContracts_AddToMempool_OnlyValid()
                 tx.Sign(stratisNodeSync.FullNode.Network, stratisNodeSync.MinerSecret, false);
                 stratisNodeSync.Broadcast(tx);
 
+                // OP_RETURN (Anything non-P2PKH
```

Comment:
I would say that we should probably test multiple different varieties instead of just the one, but as a first implementation I suppose it is ok.